src/components/email-list/EmailList.tsx

```diff
@@ -2,12 +2,15 @@ import * as React from "react";
 import { v1 as uuid } from "uuid";
 import { StaticQuery, graphql } from "gatsby";
 import { EmailListItem } from "./EmailPageLink";
+import { LetterListItem } from "./LetterListItem";
```

Comment:
I think we should change this directory structure. let's move relevant files into a components/list directory so that it's no longer solely an `EmailList` and so that `LetterListItem` doesn't live inside of a package called `email-list`

Reply:
agreed. I just ... didn't want to make this commit any larger than it was